magefile.go

```diff
@@ -987,8 +992,13 @@ func (Docker) Image() error {
 	if v, ok := os.LookupEnv(envDockerImage); ok && v != "" {
 		image = v
 	}
+	dockerEnv := map[string]string{"DOCKER_BUILDKIT": "1"}
+	if err := sh.RunWithV(dockerEnv, "docker", "buildx", "create", "--use"); err != nil {
+		return fmt.Errorf("docker buildx create failed: %w", err)
+	}
 
-	return sh.RunWithV(map[string]string{"DOCKER_BUILDKIT": "1"}, "docker", "build",
+	return sh.RunWithV(dockerEnv, "docker", "buildx", "build",
+		"--platform", strings.Join(platformsDocker, ","),
```

Comment:
should this be filtered by the PLATFORMS env var (I think with the make approach we could that)? what will happen if somebody on an x86 machine only invokes this?

Reply:
Right now when someone runs this, it will build the amd64 and arm64 images, I'll add filtering but I think that would mostly apply for developers working locally

Reply:
IMO since this was supported in the previous make approach and we do have developers that work on x86 machines I feel that such filtering might be handy

Reply:
Yeah I agree being able to filter would be nice.